fdbserver/RestoreApplier.actor.cpp

```diff
@@ -301,15 +310,25 @@ ACTOR static Future<Void> precomputeMutationsResult(Reference<ApplierBatchData>
 	    .detail("StagingKeys", batchData->stagingKeys.size());
 
 	// Get keys in stagingKeys which does not have a baseline key by reading database cx, and precompute the key's value
+	std::vector<Future<Void>> fGetAndComputeKeys;
 	std::map<Key, std::map<Key, StagingKey>::iterator> imcompleteStagingKeys;
```

Comment:
Typo: `imcompleteStagingKeys` -> `incompleteStagingKeys`

Reply:
Ah, good catch! I'm so glad (or sad) compiler does not check grammar errors. ;)